versioned_docs/version-8.7/self-managed/operational-guides/backup-restore/backup-and-restore.md

```diff
@@ -5,73 +5,1287 @@ sidebar_label: "Backup and restore"
+            }
+         ]
+      }
+      ```
+
+      </summary>
+
+   </details>
+
+Alternatively as a one-line to wait until the state is `COMPLETED` using a while loop and jq to parse the response JSON.
+
+```bash
+while [[ "$(curl -s "$OPERATE_MANAGEMENT_API/actuator/backups/$BACKUP_ID" | jq -r .state)" != "COMPLETED" ]]; do echo "Waiting..."; sleep 5; done; echo "Finished backup with ID $BACKUP_ID"
+```
+
+#### 6. Wait until the backup `x` of Tasklist is complete. Using the [Tasklist management backup API](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md)
+
+```bash
+curl "$TASKLIST_MANAGEMENT_API/actuator/backups/$BACKUP_ID"
+```
+
+   <details>
+      <summary>Example output</summary>
+      <summary>
+
+      ```json
+      {
+         "backupId":1748937221,
+         "state":"COMPLETED",
+         "failureReason":null,
+         "details":[
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_1_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:56.519+0000",
+               "failures":[
+
+               ]
+            },
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_2_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:57.324+0000",
+               "failures":[
+
+               ]
+            },
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_3_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:57.927+0000",
+               "failures":[
+
+               ]
+            },
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_4_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:58.329+0000",
+               "failures":[
+
+               ]
+            },
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_5_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:58.933+0000",
+               "failures":[
+
+               ]
+            },
+            {
+               "snapshotName":"camunda_tasklist_1748937221_8.7.2_part_6_of_6",
+               "state":"SUCCESS",
+               "startTime":"2025-06-03T07:56:59.535+0000",
+               "failures":[
+
+               ]
+            }
+         ]
+      }
+      ```
+
+      </summary>
+
+   </details>
+
+Alternatively as a one-line to wait until the state is `COMPLETED` using a while loop and jq to parse the response JSON.
+
+```bash
+while [[ "$(curl -s "$TASKLIST_MANAGEMENT_API/actuator/backups/$BACKUP_ID" | jq -r .state)" != "COMPLETED" ]]; do echo "Waiting..."; sleep 5; done; echo "Finished backup with ID $BACKUP_ID"
+```
+
+### Backup of the Zeebe Cluster
+
+#### 1. Soft pause exporting in Zeebe. Using the [management API](/self-managed/zeebe-deployment/operations/management-api.md)
```

Comment:
I know from SaaS that many customer were worried that a backup would stop processing. Should we mention that soft pause will not stop the processing (or link to the glossary what soft pause mean/ do)?

Reply:
Glossary is 1:1 the same explanation as on the linked page.

Thanks, I adjusted the fragment identifier to link directly and pre-select the right tab.
I also repeated the same thing as quick one-liner again as outline on the linked page.